src/components/nav-menu/nav-menu.js

```diff
@@ -95,135 +88,99 @@ const NavMenu = ({ width }) => {
       </ListItemIcon>
       <ListItemText primary={pathTitle} />
       {pathTitle === titles.emailQuestionsTitles.mainPageTitle && (
-        <Badge badgeContent={pendingQuestionsCount} color='error' />
+        <Badge
+          badgeContent={pendingQuestionsCount}
+          className={classes.bageRight}
+          color='error'
+        />
       )}
     </ListItem>
   );
 
-  const menuItems = config.menuCategories.map((category) =>
-    returnedList(category[0], category[1], category[2])
-  );
-
-  const materialMenuItems = config.materialMenuCategories.map((category) =>
-    returnedList(...category, classes.nested)
-  );
-
-  const clientMenuItems = config.clientMenuCategories.map((category) =>
-    returnedList(category[0], category[1], category[2], classes.nested)
-  );
-
-  const catalogMenuItems = config.catalogMenuCategories.map((category) =>
-    returnedList(category[0], category[1], category[2], classes.nested)
-  );
-
-  const promoMenuItems = config.promoMenuCategories.map((category) =>
-    returnedList(...category, classes.nested)
-  );
-
-  const staticPagesMenuItems = config.staticPagesCategories.map((category) =>
-    returnedList(category[0], category[1], category[2], classes.nested)
-  );
-
-  const constructorPagesMenuItems = config.constructorMenuCategories.map(
-    (category) =>
-      returnedList(category[0], category[1], category[2], classes.nested)
+  function makeMenuItems(arrayOfCategories, className) {
```

Comment:
Use the arrow function declaration, please.

Reply:
fixed